### PR TITLE
Add base .travis.yml from Twenty Sixteen

### PR DESCRIPTION
**Pull request by josephfusco**
WordPress Username: [joefusco](https://profiles.wordpress.org/joefusco)

### CODE REVIEW

**Comment by josephfusco** [comment]
Resolves #29 by introducing the travis config used for Twenty Sixteen development.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Travis CI configuration file.
+# @link https://travis-ci.org/
+
+# For use with the Twenty Sixteen WordPress theme
```
Comment (ntwb) [inline .travis.yml]: This should be "Twenty Seventeen"

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# Run test script commands.
+# Default is specific to project language.
+# All commands must exit with code 0 on success. Anything else is considered failure.
+script:
+    # Search theme for PHP syntax errors.
+    - find . \( -name '*.php' \) -exec php -lf {} \;
+    # Run the theme through JSHint
+    - jshint .
+    # Run the theme through JavaScript Code Style checker
+    - jscs .
```
Comment (ntwb) [inline .travis.yml:86]: There is no `.jscsrc` file currently in this pull request, either remove lines #69-#71 and #85-#86 or include the `.jscsrc` file from the Twenty Sixteen repo https://github.com/WordPress/twentysixteen/blob/master/.jscsrc

**Review by ntwb — CHANGES_REQUESTED** [review]

**Review by grappler — APPROVED** [review]

**Review by ntwb — APPROVED** [review]
Thanks @josephfusco, looks good 👍

File: .jscsrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "preset": "wordpress",
+    "fileExtensions": [ ".js" ],
+    "excludeFiles": [
+        "js/html5.js"
```
Comment (ernilambar) [inline .jscsrc]: May be `assets/js/html5.js` ?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+# Ditch sudo and use containers.
+# @link http://docs.travis-ci.com/user/migrating-from-legacy/#Why-migrate-to-container-based-infrastructure%3F
+# @link http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure
+sudo: false
+
+# Declare which versions of WordPress to test against.
+# Also declare whether or not to test in Multisite.
+env:
+    # Trunk (current version in development is 4.4)
```
Comment (ernilambar) [inline .travis.yml]: Current development version should be 4.7

**Review by ntwb — APPROVED** [review]

**Comment by josephfusco** [comment]
**Reminder**: once merged twentyseventeen will need to be added to https://travis-ci.org/wordpress/

**Comment by josephfusco** [comment]
This would be really helpful to implement ASAP with the recent surge in issues and PRs.

**Comment by ernilambar** [comment]
I am also not sure why this is on hold. This should have been in high priority in the review.

**Pull request merged**